beta/azure-container-apps/ReadMe.md

```diff
@@ -0,0 +1,233 @@
+2. Define your variables for the deployment by entering the following text on the Cloud Shell after updating it to meet your deployment:
+    ```
+    ResourceGroup="op-scim-bridge-rg"
+    Location="canadacentral"
+    ContainerAppEnvironment="op-scim-bridge-con-app-env"
+    ContainerAppName="op-scim-bridge-con-app"
+    ```
+3. Create the resource group 
+    ```
+    az group create --name $ResourceGroup --location $Location
+    ```
+4. Create the Container App Environment 
+    ```
+    az containerapp env create --name $ContainerAppEnvironment --resource-group $ResourceGroup --location $Locatio
+    ```
+5. Create the base Container App:
+    ```
+    az containerapp create -n $ContainerAppName -g $ResourceGroup --container-name op-scim-bridge \
+    --image docker.io/1password/scim:v2.8.2 --environment $ContainerAppEnvironment --ingress external \
+    --target-port 3002 --cpu 0.25 --memory 0.5Gi --min-replicas 1 --max-replicas 1 \
+    --env-vars 'OP_REDIS_URL=redis://localhost:6379' 'OP_SESSION=""'
+    ```
+    > **Note:** 
+    > 
+    >If you are prompted to enable the ContainerApp extension, with the error `The command requires the extension containerapp. Do you want to install it now? The command will continue to run after the extension is installed. (Y/n):`, type Y.
+
+6. Before you create the container Secret, upload your scimsession file to the Cloud Shell:
+   1. Click the “Upload/Download files” button and choose Upload.
+   2. Find the `scimsession` file that you saved to your computer and choose it.
+   3. Make note of the upload destination, then click Complete.
+7. Create Azure Container App Secret:
+    ```
+    az containerapp secret set -n $ContainerAppName -g $ResourceGroup --secrets scimsession="$(cat /home/$USER/scimsession | base64)"
+    ```
+8. Update your Container App with the required secondary container for Redis:
+    ```
+    az containerapp update -n $ContainerAppName -g $ResourceGroup --container-name op-scim-redis \
+    --image docker.io/redis --cpu 0.25 --memory 0.5Gi --set-env-vars 'REDIS_ARGS=--maxmemory 256mb --maxmemory-policy volatile-lru'
+    ```
+9. Update your Container App to allow your SCIM bridge container to use the secret you have created:
+    ```
+    az containerapp update -n $ContainerAppName -g $ResourceGroup --container-name op-scim-bridge \
+    --set-env-vars 'OP_REDIS_URL=redis://localhost:6379' 'OP_SESSION=secretref:scimsession'
+    ```
+10. Find your fdqn of your deployed application, this will be known as your SCIM bridge URL:
+    ```
+    az containerapp ingress show -n $ContainerAppName -g $ResourceGroup
+    ```
+11. Open the fqdn listed in a seperate browser tab, where you can test your connection. You should be prompted to log into the SCIM bridge with your `bearer token`.
+
+### Step 3: Follow the steps to connect your Identity provider to the SCIM bridge.
+ - [Connect your Identity Provider](https://support.1password.com/scim/#step-3-connect-your-identity-provider)
```

Comment:
Trying to commit Pike suggested changes

```suggestion
1. Start the Azure Cloud Shell from the navigation bar of your [Azure Portal](https://portal.azure.com).
2. Define variables for the deployment using the following example in the Cloud Shell. Update the values in a text editor before pasting it into the terminal:

   ```bash
   ResourceGroup="op-scim-bridge-rg"
   Location="canadacentral"
   ContainerAppEnvironment="op-scim-bridge-con-app-env"
   ContainerAppName="op-scim-bridge-con-app"
   ```

3. Create the resource group:

   ```bash
   az group create --name $ResourceGroup --location $Location
   ```

4. Create the Container App Environment:

   ```bash
   az containerapp env create --name $ContainerAppEnvironment --resource-group $ResourceGroup --location $Location
   ```

5. Upload your `scimsession` file to the Cloud Shell:
   - Click the “Upload/Download files” button and choose Upload.
   - Find the `scimsession` file that you saved to your computer and choose it.
   - Make a note of the upload destination, then click Complete.

6. Create the base Container App and Secret for your Base64-encoded `scimsession` credentials:

   ```bash
    az containerapp create -n $ContainerAppName -g $ResourceGroup \
        --container-name op-scim-bridge \
        --image docker.io/1password/scim:v2.8.2 \
        --environment $ContainerAppEnvironment \
        --ingress external --target-port 3002 \
        --cpu 0.25 --memory 0.5Gi \
        --min-replicas 1 --max-replicas 1 \
        --secrets scimsession="$(cat /home/$USER/scimsession | base64)" \
        --env-vars OP_REDIS_URL="redis://localhost:6379" OP_SESSION=secretref:scimsession
    ```

    > **Note**
    >
    > You may be prompted to install the ContainerApp extension with this message:
    >
    > ```bash
    > The command requires the extension containerapp. Do you want to install it now? The command will continue to run after the extension is installed. (Y/n):
    > ```
    >
    > The ContainerApp extension is required to deploy Container Apps from the command line. Type `Y` and press enter to install the extension.

7. Update your Container App to add the Redis container and get the fully qualified domain name to use as the URL for your SCIM bridge:

   ```bash
   az containerapp update -n $ContainerAppName -g $ResourceGroup \
       --container-name op-scim-redis \
       --image docker.io/redis \
       --cpu 0.25 --memory 0.5Gi \
       --set-env-vars REDIS_ARGS="--maxmemory 256mb --maxmemory-policy volatile-lru" \
       --query properties.configuration.ingress.fqdn
   ```

8. Open the domain name listed in a separate browser tab to test your connection. You can sign in to your SCIM bridge using your bearer token.
```